docs/source/open-software-week/index.md

```diff
@@ -0,0 +1,95 @@
+# Open Software Week
+
+We are excited to announce our inaugural **NIU Open Software Week**, taking
+place in **August 2025** in **London, UK**. This event will bring together researchers, developers, and users of open-source software for some hands-on training and hacking.
```

Comment:
```suggestion
place in **August 2025** in **London, UK**. This event will bring together researchers, developers, and users of open-source software for some hands-on training, community-building and hacking.
```

Reply:
While this is more explicit in the microscopy track pitch, I think community building applies to both?